### PR TITLE
preview-tui: allow pdf text preview

### PR DESCRIPTION
**Pull request by sweenu**
Closes https://github.com/jarun/nnn/issues/1283

### CODE REVIEW

File: plugins/preview-tui [hunk 1]
```diff
@@ -246,7 +251,11 @@ handle_mime() {
 handle_ext() {
     case "$2" in
         epub) generate_preview "$cols" "$lines" "$1" "epub" ;;
-        pdf) generate_preview "$cols" "$lines" "$1" "pdf" ;;
+        pdf) if [ "$PDFTOTEXT" = 0 ]; then
```
Comment (N-R-K) [inline plugins/preview-tui:254]: Should use `-eq` instead.
```suggestion
        pdf) if [ "$PDFTOTEXT" -eq 0 ]; then
```

**Comment by luukvbaal** [comment]
It is recommended to use `preview-tui` with one of the image preview options that displays images in original quality.

I would say original quality pdf preview for the first page is more useful than `pdftotext`. If you do prefer `pdftotext` I would suggest to make the change locally. We can't make an envvar for each change in previewer I would say.

**Comment by sweenu** [comment]
It's up to you, but the way I see it, preview-tui is a genric preview plugin, not only for images but also for texts, markdown, code, etc. so many might want to use the plugin for its text previews primarily, not caring for the images.
I already made the change locally, you can go ahead and close this if you think no one else would want that :)

**Comment by luukvbaal** [comment]
I'm not saying no one else would want that, but I don't see the point in adding a new preview option locked behind an envvar. Plugins are indicative.

**Pull request closed without merging**